config/bidderinfo_validate_test.go

```diff
@@ -54,7 +55,16 @@ func TestBidderInfoFiles(t *testing.T) {
 
 		err = validateInfo(&fileInfoContent)
 		assert.NoError(t, err, "Invalid content in static/bidder-info/%s: %v", fileInfo.Name(), err)
+
+		err = validateSyncer(fileInfoContent)
+		assert.NoError(t, err, "Invalid syncer config in static/bidder-info/%s: %v", fileInfo.Name(), err)
+
+		fileNameWithoutExtension := fileInfo.Name()[:len(fileInfo.Name())-len(filepath.Ext(fileInfo.Name()))]
+		bidderInfos[fileNameWithoutExtension] = fileInfoContent
 	}
+
+	errs := validateSyncers(t, bidderInfos)
+	assert.Empty(t, errs, "syncer errors")
 }
```

Comment:
I improved the unit tests to use the same exact loading mechanism the app uses at startup to ensure no issues / the same validation is performed.